DevCycle/DVCClient.swift

```diff
@@ -267,14 +267,17 @@ public class DVCClient {
             // TODO: save config in cache
             self.user = user
             self.cacheService.save(user: user)
-            self.cacheService.clearAnonUserId()
+            if (!(user.isAnonymous ?? false)) {
```

Comment:
can we simplify this to be:
```suggestion
            if let isAnonymousUser = user.isAnonymous, !isAnonymousUser {
```